### ISSUE TITLE
Cleanup cluster state on startup

### ISSUE THREAD

**Issue by cfergeau** [code-ready/crc#336]
currently right after starting the crc VM, when checking the state of pods, some of them could appear as properly running while it's just left-over state from the previous VM run which is not properly cleaned up on shutdown. We should clean this up when starting up the VM before starting the kubelet. This probably involves changing some values in etcd (?)

**Comment by gbraad** [comment]
Still relevant... what would be possible?

At the moment we try to remove pods and have them recreated, but still references to the initial run exist.

**Comment by gbraad** [comment]
this is something we can't fix until manually change the etcd data, #wontfix
